contrib/pyln-spec/bolt1/pyln/spec/bolt1/text.py

```diff
@@ -10,7 +10,7 @@
 
 - Bitcoin mainet with port number 9735 or the corresponding hexadecimal `0x2607`;
 - Bitcoin testnet with port number 19735 (`0x4D17`);
-- Bitcoin signet with port number 39735 (`0xF87`).
+- Bitcoin signet with port number 39735 (`0x9B37`).
```

Comment:
nit: I think this file is autogenerated from a bolt version

Reply:
Indeed it is, but until upstream is fixed we can just keep that here, and fixing upstream will obviate this diff further down the line.